pkg/operator/rbaccontroller/rbaccontroller.go

```diff
@@ -0,0 +1,90 @@
+package rbaccontroller
+
+import (
+	"context"
+	"strings"
+
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	errors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apiserver/pkg/authentication/user"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	// Bug 1821771: We needed a way to tighten the cluster role binding when cluster
```

Comment:
we usually don't link BZs in code. PR description is enough.

Reply:
Done. Removed the link to BZ